### ISSUE TITLE
query/data issues

### ISSUE THREAD

**Issue by ben-williams** [commfish/seak_sablefish#2]
Survey CPUE (survey_cpue_1988_2016.csv)

Current query lacks:
1) depth  - will need to run depth standardized and status quo cpue to provide to managers
2) survey station and actual lat/lon

**Comment by jysullivan** [comment]
It's on the list.

**Comment by jysullivan** [comment]
Also survey station lat/lon

**Comment by jysullivan** [comment]
Longline (LL) survey biological data  - there were two csvs in Kray's files. 
1.  'srv_bio_data' - very detailed that includes time stamp, setstart/end lat/lon, depth, etc. from 1998 to 2015.
2. 'srv_bio_data2' - minimal, no time stamp, lat/lon, depth from 1988 to 2016.

Use more detailed query in 2017. Also get station lat/lon along with set start/end lat/lon. @aaronpbaldwin can you please make a comment here about the discussion we had about station definitions on the NSEI LL survey?

Need:
1) An explanation of 'Effort No'

**Comment by jysullivan** [comment]
fishery biological data (fishery_bio_2000_2016.csv): 

ALEX QUERY CRITERIA FOR FISH_BIO_DATA
Year BETWEEN 1980 AND 2015 AND species_code = '710' AND 
g_management_area_code = 'NSEI' AND project_code = '02'

KVK NOTE: when running, always make sure that all data have been read and uploaded into IFDB by the ADU

Need an explanation of 
1) G_STAT_AREA_GROUP (e.g. NSEI 22, NSEI 15)
2) SPECIMEN_NO 
3) SAMPLE_TYPE_CODE
4) WEIGHT_KILOGRAMS (round? ice/slime? some of these are really heavy, e.g. 17 kg)

**Comment by jysullivan** [comment]
Lengths - what are the length type code definitions? is there an established conversion between total and fork length?

Weights - when are they dressed, round, ice/slimed? conversion factors?

Maturity code definitions

Age readability codes

Release condition codes

Discard status codes

Depth - I have some of these coming to me as meters and others as fathoms

**Comment by jysullivan** [comment]
Pot survey data 'potsurvey_bio_2009_2015.csv'

Only have age/sex data for 2009. 

Aaron Baldwin 2017-10-24
> Up until 2010 we collected biological data on a subset of sablefish. 
> On the marking survey all fish are measured (when possible), even fish not tagged due to injury or mortality. The only real exception are when pots are dumped at rail due to reaching tagging goal. There are always a few that sneak through, either they escape prior to measuring or the recorder misses the length. But procedurally all sablefish brought onboard are measured regardless of being tagged or not.

There are three length entries that measured to the nearest mm instead of cm. Errors?

**Comment by aaronpbaldwin** [comment]
Yes, we record all lengths to nearest cm but lengths are recorded in mm. For example, a 74 cm fish is recorded as 740 mm. There are still errors (especially with port samplers as we often borrow seasonal techs used to salmon which are measured to the nearest 5 mm).

**Comment by jysullivan** [comment]
@aaronpbaldwin Hi Aaron, do you use OceanAK to pull data or do you query the databases directly?

**Comment by aaronpbaldwin** [comment]
I use OceanAK to pull data.

**Comment by jysullivan** [comment]
Great, thanks. If I understand correctly users can store and save 'canned reports'. I'd like to translate these to sql so that I can query the databases directly from inside R. Do you have OceanAK reports saved that you could share with me? I have some stuff from Kray, which I've documented in the README.md file (access this by clicking on the 'code' tab at the top of the page and scrolling down)
